elixir-mode.el

```diff
@@ -11,7 +11,7 @@
 ;; Created: Mon Nov 7 2011
 ;; Keywords: languages elixir
 ;; Version: 2.4.0
-;; Package-Requires: ((emacs "24") (pkg-info "0.4"))
+;; Package-Requires: ((emacs "24") (pkg-info "0.4") (s "1.2.0"))
```

Comment:
Since `s` is only used for tests this should be changed so it's not a general requirement of the package. Unfortunately I wasn't able to find any information on how to accomplish that

Reply:
Doesn't that help? https://github.com/doublep/eldev#additional-dependencies